app/src/main/kotlin/com/simplemobiletools/notes/activities/MainActivity.kt

```diff
@@ -87,6 +97,34 @@ class MainActivity : SimpleActivity(), ViewPager.OnPageChangeListener {
         }
     }
 
+    private fun handleFile(path: String) {
+        val note = mDb.getNote(path)
+
+        if (note != null) {
+            updateSelectedNote(note.id)
+            return
+        }
+
+        var title = path.getFilenameFromPath()
+
+        if (mDb.doesTitleExist(title)) title += " (file)"
+
+        val openFileFunc = {
```

Comment:
make it a normal function or solve it some other way, this looks really weird and unusual